<ui version="4.0" >
 <class>ChannelWidget</class>
 <widget class="QWidget" name="ChannelWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>519</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>3</hsizetype>
    <vsizetype>3</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>200</width>
    <height>100</height>
   </size>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>YourNickname #quassel Network: The Topic</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>1</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="topicEdit" >
       <property name="focusPolicy" >
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="chanSettingsButton" >
       <property name="windowModality" >
        <enum>Qt::NonModal</enum>
       </property>
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection" >
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="lineWidth" >
      <number>1</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTextBrowser" name="chatWidget" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>4</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font" >
       <font>
        <family>Bitstream Vera Sans Mono</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="html" >
       <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans Mono'; font-size:8pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Nimbus Mono L';">&lt;/p>&lt;/body>&lt;/html></string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="nickTree" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection" >
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="dragEnabled" >
       <bool>true</bool>
      </property>
      <property name="textElideMode" >
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="rootIsDecorated" >
       <bool>true</bool>
      </property>
      <property name="sortingEnabled" >
       <bool>false</bool>
      </property>
      <property name="animated" >
       <bool>false</bool>
      </property>
      <property name="columnCount" >
       <number>1</number>
      </property>
      <column>
       <property name="text" >
        <string>32 Users</string>
       </property>
      </column>
      <item>
       <property name="text" >
        <string>@ Operators</string>
       </property>
       <item>
        <property name="text" >
         <string>New Sub Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text" >
        <string>4 Voiced</string>
       </property>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
      </item>
      <item>
       <property name="text" >
        <string>19 Users</string>
       </property>
       <item>
        <property name="text" >
         <string>New Sub Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>New Item</string>
        </property>
       </item>
      </item>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QComboBox" name="ownNick" >
       <property name="sizeAdjustPolicy" >
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ChannelWidgetInput" name="inputEdit" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChannelWidgetInput</class>
   <extends>QLineEdit</extends>
   <header>channelwidgetinput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>inputEdit</tabstop>
  <tabstop>ownNick</tabstop>
  <tabstop>nickTree</tabstop>
  <tabstop>topicEdit</tabstop>
  <tabstop>chanSettingsButton</tabstop>
  <tabstop>chatWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
